examples/custom-component/src/hooks/after_incoming_middleware/sendoptions.js

```diff
@@ -0,0 +1,11 @@
+// Simply write 'options' when talking with the bot to receive the drop down menu
+if (event.preview === 'options') {
```

Comment:
I imagine that can be done by an action as well ...  IMO that would make more sens as options could be configurable in the studio UI

Reply:
Yeah, i just made this real quick to test